src/main/java/seedu/address/logic/commands/FilterCommand.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.tag.ElderlyHasTagPredicate;
+
```

Comment:
I think it would be better if the class had a header comment? (since the ip feedback for documentation mentions class header comments)

Reply:
Okay I will add it!